Makefile.am

```diff
@@ -329,12 +329,12 @@ tests-pylint:
 	$(MAKE)
 	$(MAKE) TMPDIR=/var/tmp TEST_SUITE_LOG=test-suite.log TESTS=pylint/runpylint.py check
 
-tests-nose-only:
+tests-units-only:
```

Comment:
I have never seen "units" as a plural for tests. Feel free to prove me wrong, though!

Reply:
Fixed.